lms/djangoapps/teams/static/teams/js/views/topic_card.js

```diff
@@ -1,52 +1,62 @@
 /**
  * View for a topic card. Displays a Topic.
  */
-;(function (define) {
+;(function(define) {
     'use strict';
-    define(['backbone', 'underscore', 'gettext', 'js/components/card/views/card'],
-        function (Backbone, _, gettext, CardView) {
+    define([
+            'backbone', 'underscore', 'gettext', 'js/components/card/views/card',
+            'edx-ui-toolkit/js/utils/html-utils',
+            'edx-ui-toolkit/js/utils/string-utils'
+        ],
+        function(Backbone, _, gettext, CardView, HtmlUtils, StringUtils) {
             var TeamCountDetailView = Backbone.View.extend({
                 tagName: 'p',
                 className: 'team-count',
 
-                initialize: function () {
+                initialize: function() {
                     this.render();
                 },
 
-                render: function () {
-                    var team_count = this.model.get('team_count');
-                    this.$el.html(_.escape(interpolate(
-                        ngettext('%(team_count)s Team', '%(team_count)s Teams', team_count),
-                        {team_count: team_count},
-                        true
-                    )));
+                render: function() {
+                    var teamCount = this.model.get('team_count');
+                    HtmlUtils.setHtml(
```

Comment:
@robrap Here's an example where we use ``HtmlUtils.setHtml`` with a string. It seems useful for updating simple paragraph and span tags. I couldn't find an example for ``HtmlUtils.append`` but it seems reasonable to support it for consistency.